modules/compute/gke-node-pool/variables.tf

```diff
@@ -433,3 +433,9 @@ variable "enable_private_nodes" {
   type        = bool
   default     = false
 }
+
+variable "num_node_pools" {
+  description = "Number of node pools to create. For TPUs, this is the number of slices."
```

Comment:
Use a separate variable for number of slices as that's the default TPU nomenclature. Can be addressed in #3817